docs/UserGuide.md

```diff
@@ -97,13 +97,24 @@ Updated! Number of rows affected: 1
 ```
 
 ### Listing medication stock : `list`
+Lists all existing medication in the inventory.
+* All parameters for list command are optional, you can choose to list medication by any of the parameters.
+* The following example demonstrates list by medication name.
 
-Format:
+Format: `list {i/stock_id n/name p/price q/quantity e/expiry_date d/description m/max_quantity sort/column_name rsort/column_name} `
 
-Example:
+Example:  `list n/panadol`
```

Comment:
Add another example and expected output that is just `list` without parameters. Put that as example 1. You can refer to Deon's add medication

Reply:
I have fixed the issues